admin_manual/installation/changing_the_web_route.rst

```diff
@@ -1,48 +1,23 @@
-======================
-Changing the Web Route
-======================
+==========================
+Changing Your ownCloud URL
+==========================
 
-This admin manual assumes that the owncloud server shall be accessible under the web route
-``/owncloud`` -- this is also where the Linux packages make the server appear. You can change this in your Web server configuration, for example from ``https://example.com/owncloud/`` to ``https://example.com/``.
+This admin manual assumes that the ownCloud server is already accessible under the Web route
+``/owncloud``, which is the default at installation, e.g. ``https://example.com/owncloud``. You can change this in your Web server configuration, for example from ``https://example.com/owncloud/`` to ``https://example.com/``. 
 
-Basic system administrator and Apache configuration knowledge is prerequisite.
-Several configuration files need to be kept in sync when changing the Web route location.
+On Debian/Ubuntu Linux edit these files:
 
-On an Ubuntu-14.04 system the following files are typically involved:
-
-- ``/etc/apache2/conf-enabled/owncloud.conf``
+- ``/etc/apache2/sites-enabled/owncloud.conf``
```

Comment:
Ubuntu 14.04 systems (and also Debian) is still installing to /etc/apache2/conf-enabled/owncloud.conf and not to /etc/apache2/sites-enabled/owncloud.conf